charts/nfs-subdir-external-provisioner/values.yaml

```diff
@@ -61,6 +61,11 @@ rbac:
   # Specifies whether RBAC resources should be created
   create: true
 
+## For SCC support (Openshift):
+scc:
```

Comment:
when changing the helm chart values/template, it is required to bump the helm chart version in the `Chart.yaml` as well, unless, the PR will be un-mergable since the tests will fail because every change in the helm chart triggers automatic chart version release once merged to `master`.